components/service/nimbus/src/main/java/mozilla/components/service/nimbus/Nimbus.kt

```diff
@@ -67,7 +68,9 @@ interface NimbusApi {
 private const val LOG_TAG = "service/Nimbus"
 private const val EXPERIMENT_BUCKET_NAME = "main"
 private const val EXPERIMENT_COLLECTION_NAME = "nimbus-mobile-experiments"
-private const val NIMBUS_DATA_DIR: String = "nimbus_data"
+private const val NIMBUS_DATA_DIR = "nimbus_data"
+private const val MAX_RETRY_COUNT = 3
+private const val RETRY_DELAY_MS = 10000L
```

Comment:
This is now dead code.

```suggestion
```

I for one am very please you're not doing the retries in this class.